apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentReadResourceIntegrationTest.java

```diff
@@ -2631,7 +2631,12 @@ public void escapedChar() {
           .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
           .queryParam("raw", true)
           .when()
-          .get(BASE_PATH + "/{document-id}/a%5C.b", DEFAULT_NAMESPACE, DEFAULT_COLLECTION, id)
+          .get(
+              BASE_PATH + "/{document-id}/{leaf}",
+              DEFAULT_NAMESPACE,
+              DEFAULT_COLLECTION,
+              id,
+              "a\\.b")
```

Comment:
In this case need first doubling for Java compiler,  but then `\` itself handled by Docs API server side to avoid interpreting following `.` as path separator.